versioned_docs/version-1.3/components/best-practices/architecture/deciding-about-your-stack-c7.md

```diff
@@ -7,13 +7,13 @@ tags:
     - Application Server
     - Spring Boot
     - Maven
-description: "Camunda Platform 7 is very flexible and can be hooked into the architecture of your choice, giving you a number of important decisions to make."
+description: " 7 is very flexible and can be hooked into the architecture of your choice, giving you a number of important decisions to make."
```

Comment:
🪦 Oops

```suggestion
description: "Camunda 7 is very flexible and can be hooked into the architecture of your choice, giving you a number of important decisions to make."
```

Looks like this impacted all the 1.3 docs. Can you have a look?

Reply:
🔴 Adjusted!